app-devel/lfortran/autobuild/defines

```diff
@@ -0,0 +1,22 @@
+PKGNAME=lfortran
+PKGSEC=devel
+PKGDEP="gcc-runtime glibc llvm ncurses python-3"
+PKGDES="Interactive LLVM-based Fortran compiler"
+
+BUILDDEP="fmt llvm python-3 re2c zlib-static zstd-static"
```

Comment:
Weird placement, can you put this under PKGDEP so it won't be so easy to miss?

Reply:
Also, why were the static libraries needed?

Reply:
The cmake file specifically asks for the static library. e.g., whith `zlib` instead of `zlib-static`, we still get the following error:
```bash
...
CMake Error at /usr/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find ZLIB (missing: ZLIB_LIBRARY) (found version "1.3.1")
Call Stack (most recent call first):
  /usr/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.31/Modules/FindZLIB.cmake:202 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cmake/FindStaticZLIB.cmake:7 (find_package)
  CMakeLists.txt:124 (find_package)
...
```

`BUILDDEP` has been fixed.